app/views/posts/show.html.erb

```diff
@@ -1 +1,25 @@
-<h2>Welcome to posts show page</h2>
\ No newline at end of file
+<h1>Welcome to posts show page</h1>
+<section class="post_card column">
+  <div class="post_detail_title">
+    <h3 class="margin">Post #<%= 1 %></h3>
+    <div class="margin comments_likes">
+    <h4>Comments: <%= @post.comments_counter%></h4>
+    <h4>Likes: <%= @post.likes_counter%></h4>
+    </div>    
+  </div>
+  <div class="margin"> 
+    <p><%= @post.text%> </p>
+  </div>
+  
+</section>
+<section class="post_card comments_card">
+    <% comments = @post.recent_comments %>
```

Comment:
- [ ] Hi @dfsalam, on this post show page, we are not supposed to filter comments. We should display all user's comments here :smile: It should be like the user's details page where we can find all the information of a user hence we shouldn't use the @recent_comments method here

Reply:
Fixed.